app/Http/Controllers/Wiki/CountriesController.php

```diff
@@ -58,15 +58,18 @@ public function index()
         $page      = checkParam('page') ?? 1;
 
         list($limit, $is_bibleis_gideons) = forceBibleisGideonsPagination($this->key, $limit);
+
         $cache_params = [$GLOBALS['i18n_iso'], $languages, $limit, $page, $is_bibleis_gideons];
+        $cache_key = generateCacheSafeKey('countries_list', $cache_params);
 
-        $countries = cacheRemember('countries', $cache_params, now()->addDay(), function () use ($languages, $limit, $page) {
+        $countries = cacheRememberByKey($cache_key, now()->addDay(), function () use ($languages, $limit) {
```

Comment:
remind me why the page is not used. I believe the intent is to paginate the countries response

Reply:
@bradflood , the page value is used. But In this case, the paginate method reads the `page` value directly from the HTTP request so that, it is not necessary to pass `page` as parameter. You can see it on the Laravel documentation:
https://laravel.com/docs/9.x/pagination#paginating-query-builder-results

```php
// app/Http/Controllers/Wiki/CountriesController.php
$countries = Country::with('currentTranslation')
   ...
    ->paginate($limit);
```